blockchain/vm/internaltx_tracer.go

```diff
@@ -267,7 +267,7 @@ func (this *InternalTxTracer) step(log *tracerLog) error {
 
 		// Skip any pre-compile invocations, those are just fancy opcodes
 		toAddr := common.HexToAddress(log.stack.Back(1).Text(16))
-		if common.IsPrecompiledContractAddress(toAddr) {
+		if _, ok := PrecompiledContractsCypress[toAddr]; ok {
```

Comment:
Is there any problem for collecting data from Baobab?

Reply:
https://baobab.scope.klaytn.com/tx/0xda47fd05ece0fc7267f88a5cbcc595603add75b77287c1f3f6c48db0a0250c59?tabId=internalTx
The last two calls weren't collected. It would be better to change `callTracer` to filter the range of the precompiled contracts.